frontend/src/routes/(dashboards)/admin/+page.svelte

```diff
@@ -21,11 +21,13 @@
 <main>
 	<div class="grid grid-cols-2">
 		<div class="overflow-x-auto">
-			<span class="text-xl">{ t('admin_dashboard.page_header') }</span>
+			<span class="text-xl">
+				{ t('admin_dashboard.page_header') }
+			</span>
 
 			<Input
 				type="text"
-				label={ t('admin_dashboard.filter_label') }
+				label={ t('admin_dashboard.filter_label', {val: (123)}) }
```

Comment:
man I'm just missing stuff, please ignore this change, it shouldn't have been made.